core/hash/shared/replace.rb

```diff
@@ -1,51 +0,0 @@
-describe :hash_replace, shared: true do
-  it "replaces the contents of self with other" do
```

Comment:
Yeah, makes sense. This shared example was used also for `Hash#initialize_copy` but in d3f83d131c611ec7e8d197bf544bb61627d726d2 specs for `Hash#initialize_copy` were removed.